.buildkite/e2e-pipeline.yml

```diff
@@ -0,0 +1,14 @@
+# yaml-language-server: $schema=https://raw.githubusercontent.com/buildkite/pipeline-schema/main/schema.json
+
+agents:
+  provider: "gcp"
+  machineType: "n1-standard-4"
+  image: family/core-ubuntu-2204
```

Comment:
Should we use one of the Logstash images instead e.g. `family/platform-ingest-logstash-ubuntu-2204` so that we can benefit from preinstalled things like `ruby`, `java` and tools like `retry` etc?

Reply:
Definitely, this is I wanted to clarify with you. I saw we have two GCP & AWS similar images. I have updated in a separated [PR](https://github.com/elastic/logstash-filter-elastic_integration/pull/125)